setup.py

```diff
@@ -93,48 +94,136 @@ def get_scm_config():
 
     return {}  # use the default config
 
+HPY_EXTRA_SOURCES = [
+    'hpy/devel/src/runtime/argparse.c',
+    'hpy/devel/src/runtime/buildvalue.c',
+    'hpy/devel/src/runtime/helpers.c',
+]
+
+HPY_CTX_SOURCES = [
+    'hpy/devel/src/runtime/ctx_bytes.c',
+    'hpy/devel/src/runtime/ctx_call.c',
+    'hpy/devel/src/runtime/ctx_capsule.c',
+    'hpy/devel/src/runtime/ctx_err.c',
+    'hpy/devel/src/runtime/ctx_module.c',
+    'hpy/devel/src/runtime/ctx_object.c',
+    'hpy/devel/src/runtime/ctx_type.c',
+    'hpy/devel/src/runtime/ctx_tracker.c',
+    'hpy/devel/src/runtime/ctx_listbuilder.c',
+    'hpy/devel/src/runtime/ctx_tuple.c',
+    'hpy/devel/src/runtime/ctx_tuplebuilder.c',
+]
+
+HPY_INCLUDE_DIRS = [
+    'hpy/devel/include',
+    'hpy/universal/src',
+    'hpy/debug/src/include',
+]
+
+HPY_EXTRA_LIB_NAME = "hpyextra"
+HPY_CTX_LIB_NAME = "hpyctx"
```

Comment:
if I understand correctly, this is a cpython-specifc library, isn't it? Then, I propose to call it `hpy-extra-cpython` or something like that, which makes it crystal clear what is the intended usage

Reply:
Right. `hpyextra` is right now just for `universal` and `hpyctx` (which contains the extra sources as well) is just used for `cpython`.